Assets/UI/Panels/citypaneloverview.lua

```diff
@@ -1178,6 +1266,29 @@ end
 -- ===========================================================================
 function OnShutdown()
     LuaEvents.GameDebug_AddValue(RELOAD_CACHE_ID, "m_isShowingPanel", m_isShowingPanel);
+
+    LuaEvents.Tutorial_ResearchOpen.Remove(OnClose);
```

Comment:
adding in the Remove calls that appear in the unmodified Firaxis version of this file in the OnShutdown function